packages/@okta/vuepress-site/docs/guides/implement-password/overview/index.md

```diff
@@ -2,7 +2,9 @@
 title: Overview
 ---
 
-The Resource Owner Password Flow is intended for use cases where you control both the client application and the resource that it is interacting with. At a high-level, this flow has the following steps:
+The Resource Owner Password Flow is rarely the recommended approach. It is intended for application's for which no other flow works, as it requires your application code to be fully trusted and protected from credential-stealing attacks. It is made available primarily to provide a consistent and predictable integration pattern for legacy applications which you can't otherwise be updated to a more secure flow such as Authorization Code Flow. This should be your last option, not your first choice.
```

Comment:
- Change "which you can't otherwise be updated" to "which you can't otherwise update"